docs/ros/ros/installation/controller.mdx

```diff
@@ -6,24 +6,15 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
-Joystick controllers are used for teleoperation; they allow you to remotely drive the Jackal, whether it is a physical Jackal robot,
-or a simulated Jackal. The following instructions below detail how to pair different controllers to the Jackal’s computer; however,
+Joystick controllers are used for teleoperation; they allow you to remotely drive the robot, whether it is a physical robot,
+or a simulated robot. The following instructions below detail how to pair different controllers to the robot's computer; however,
 these instructions can also be used to pair these controllers to your own computer.
 
-### Logitech F710 Controller
```

Comment:
There might be Husky A200s with these.

Reply:
There definitely are; OPG03 shipped with one relatively recently (though it specifically is unlikely to get upgraded to Humble anytime soon).  All the older Kinetic/Melodic era robots that shipped with the 710 controllers may see customers upgrading, especially if they were sold to research labs or universities.

Reply:
We still support them in the robot.yaml but we are considering them deprecated now. There isn't really any setup for them anyways.

https://github.com/clearpathrobotics/cpr-documentation/pull/121#discussion_r1240145145